spec/services/custom_fields/hierarchy/hierarchical_item_service_spec.rb

```diff
@@ -63,39 +65,99 @@
           .to receive(:create)
                 .and_return(instance_double(CustomField::Hierarchy::Item, persisted?: false, errors: "some errors"))
 
-        result = service.generate_root
+        result = subject.generate_root
         expect(result).to be_failure
       end
     end
   end
 
   describe "#insert_item" do
     let(:custom_field) { create(:custom_field, field_format: "hierarchy", hierarchy_root: parent) }
-    let(:service) { described_class.new(custom_field) }
 
     let(:parent) { create(:hierarchy_item) }
     let(:label) { "Child Item" }
     let(:short) { "Short Description" }
 
     context "with valid parameters" do
       it "inserts an item successfully without short" do
-        result = service.insert_item(parent:, label:)
+        result = subject.insert_item(parent:, label:)
         expect(result).to be_success
       end
 
       it "inserts an item successfully with short" do
-        result = service.insert_item(parent:, label:, short:)
+        result = subject.insert_item(parent:, label:, short:)
         expect(result).to be_success
       end
     end
 
     context "with invalid item" do
       it "fails to insert an item" do
-        allow(CustomField::Hierarchy::Item)
-          .to receive(:create).and_return(instance_double(CustomField::Hierarchy::Item,
-                                                          persisted?: false, errors: "some errors"))
+        # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
❓ This cop should not trigger, as `instance_double` is a verified double?

Reply:
But the wrapping double isn't. I'll see if I can remove this by adding the correct double here.

Reply:
Fixed.